.Rbuildignore

```diff
@@ -18,3 +18,4 @@
 ^\.github$
 ^TAGS$
 ^\.httr-oauth$
+^vignettes/runif-via-pmap\.Rmd$
```

Comment:
Why are you build-ignoring this vignette?

Reply:
I'm adding it as an article and not as vignette. I would like to be added to purrr website

Reply:
Is there some reason it shouldn't be a vignette?

Reply:
I turned it into a vignette but there was a comment by Jenny that this issue of iterating over rows shouldn't be made a vignette because it relates to many packages https://github.com/tidyverse/purrr/issues/610#issuecomment-451248219 but I think this vignette is just about purrr